hacktoberfest.adoc

```diff
@@ -18,8 +18,6 @@ https://hacktoberfest.com/[Hacktoberfest] takes places annually during the month
 
 2. Submit 4 approved PRs against participating project repositories during the month of October
 
-3. Choose whether you’d like to receive a Hacktoberfest t-shirt or have a tree planted in your name
```

Comment:
@PrarthonaPaul Just a minor, it is still possible to have the tree planted in your name, see:

> The first 50,000 participants to have their first PR/MR accepted will have a tree planted in their name through Tree Nation
> Participants with four pull/merge requests accepted between October 1 and October 31 will receive a unique digital reward

So we can say: You'll have the option to have a tree planted in your name and a unique digital reward as a token of appreciation.